azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/CallingCompositeBaselineUiTest.kt

```diff
@@ -21,6 +22,8 @@ import org.junit.runner.RunWith
 class CallingCompositeBaselineUiTest {
     companion object {
         private var acsToken = ""
+        private const val groupId = "74fce2c0-520f-11ec-97de-71411a9a8e13"
```

Comment:
We can use pipeline to set groupId and teams meeting URL. We do not need to add these values in code.

Reply:
done, thanks for the feedback